sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryVersion.java

```diff
@@ -9,10 +9,6 @@
  * Schema registry service version.
  */
 public enum SchemaRegistryVersion implements ServiceVersion {
-    /**
-     * 2017 version
-     */
-    V2017_04("2017-04"),
```

Comment:
curious: why can we remove it?

Reply:
Since this is the first time it is going GA and the first time it's been through the swagger API review board there are breaking changes in the spec so that the generated URLs would not align with what this client was generated for.

Reply:
So would that mean it would no longer support `V2017_04("2017-04")` version going forward and that is an accepted breaking change?